src/main/java/frc/robot/RobotContainer.java

```diff
@@ -153,10 +153,7 @@ private void configureBindings() {
       
     }, swerveDrive));
 
-    fieldCentricButton.onTrue(new InstantCommand((
-      ) -> {
-        swerveDrive.toggleFieldCentric();
-      }, swerveDrive));
+    fieldCentricButton.onTrue(swerveDrive.toggleFieldCentric());
```

Comment:
Did you test if this works? Wouldn't the correct syntax omit the parentheses after `swerveDrive.toggleFieldCentric`, i.e. you're passing the function itself rather than the result of calling it immediately?

Reply:
I may be completely wrong, haven't written Java in a while

Reply:
Yeah we tested it and it works, so I think that that syntax works fine

Reply:
@ErikBoesen calling this function returns a command instance. That command instance is stored for use when the trigger actually becomes true